src/node/node_state.h

```diff
@@ -769,6 +762,12 @@ namespace ccf
         {
           startup_snapshot_info->is_evidence_committed = true;
         }
+
+        // Inform snapshotter of all signature entries so that this node can
+        // continue generating snapshots at the correct interval once the
+        // recovery is complete
+        snapshotter->requires_snapshot(ledger_idx);
```

Comment:
requires_snapshot() doesn't seem like the right method name here...

Reply:
Updated to `record_committable`.